File: app/src/main/java/com/bachors/iptv/models/PlaylistData.java
package com.bachors.iptv.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PlaylistData {

    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("link")
    @Expose
    private String link;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

}

File: app/src/main/java/com/bachors/iptv/models/PlaylistData.kt
package com.bachors.iptv.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class PlaylistData(
    @SerializedName("title")
    @Expose
    var title: String = "",

    @SerializedName("link")
    @Expose
    var link: String = "",

    @SerializedName("channel")
    @Expose
    var channel: String = ""
)